pkg/interpreter/simple/simple.go

```diff
@@ -34,9 +35,9 @@ type Interpreter struct {
 func New() *Interpreter {
 	return &Interpreter{
 		FrameworkPrefix: "%framework=",
-		WorkerPrefix:    "%worker=",
-		PSPrefix:        "%ps=",
-		MasterPrefix:    "%master=",
+		WorkerPrefix:    "kf_worker=",
```

Comment:
I think there is no need to change % to kf_ since % is the convention in the community.